sdk/sqlmanagement/Azure.ResourceManager.Sql/src/Generated/Models/MinimalTlsVersion.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.Sql.Models
+{
+    /// <summary> Minimal TLS version. Allowed values: 'None', 1.0', '1.1', '1.2', '1.3'. </summary>
+    public readonly partial struct MinimalTlsVersion : IEquatable<MinimalTlsVersion>
+    {
+        private readonly string _value;
+
+        /// <summary> Initializes a new instance of <see cref="MinimalTlsVersion"/>. </summary>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        public MinimalTlsVersion(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string NoneValue = "None";
+        private const string One0Value = "1.0";
+        private const string One1Value = "1.1";
+        private const string One2Value = "1.2";
+        private const string One3Value = "1.3";
+
+        /// <summary> None. </summary>
+        public static MinimalTlsVersion None { get; } = new MinimalTlsVersion(NoneValue);
+        /// <summary> 1.0. </summary>
+        public static MinimalTlsVersion One0 { get; } = new MinimalTlsVersion(One0Value);
+        /// <summary> 1.1. </summary>
+        public static MinimalTlsVersion One1 { get; } = new MinimalTlsVersion(One1Value);
+        /// <summary> 1.2. </summary>
+        public static MinimalTlsVersion One2 { get; } = new MinimalTlsVersion(One2Value);
+        /// <summary> 1.3. </summary>
+        public static MinimalTlsVersion One3 { get; } = new MinimalTlsVersion(One3Value);
```

Comment:
we need to introduce customized code to change the name of these properties, the generator cannot handle them properly.

Reply:
Please introduce a partial class of this type, and put the following code (copy the generated code here and modify the part we need):
```csharp
public partial struct MinimalTlsVersion
{
	/// <summary> 1.0. </summary>
	[CodeGenMember("One0")]
        public static MinimalTlsVersion Tls1_0 { get; } = new MinimalTlsVersion(Tls1_0Value);
        /// <summary> 1.1. </summary>
	[CodeGenMember("One1")]
        public static MinimalTlsVersion Tls1_1 { get; } = new MinimalTlsVersion(Tls1_1Value);
        /// <summary> 1.2. </summary>
	[CodeGenMember("One2")]
        public static MinimalTlsVersion Tls1_2 { get; } = new MinimalTlsVersion(Tls1_2Value);
        /// <summary> 1.3. </summary>
	[CodeGenMember("One3")]
        public static MinimalTlsVersion Tls1_3 { get; } = new MinimalTlsVersion(Tls1_3Value);
}
```
then regenerate the whole library.
Do not worry if there is compilation errors when you wrote this, it should compile after you added this class and regenerate.